docs/usage.md

```diff
@@ -135,9 +135,18 @@ If unique molecular identifiers were used to prepare the library, add the follow
 
 Please refer to the [nf-core website](https://nf-co.re/usage/reference_genomes) for general usage docs and guidelines regarding reference genomes.
 
+:::warning
+When using the --genome parameter (e.g. --genome GRCh37), you are referring to references from AWS-iGenomes. Please be aware that:
+
+- The igenomes file usage triggered by this option is outdated with respect to gene annotations. This can be particularly problematic for RNA-seq analysis, which relies on accurate gene annotation.
+- Some iGenomes references (e.g., GRCh38) point to annotation files that use gene symbols as the primary identifier. This can cause issues for downstream analysis, such as the nf-core [differential abundance workflow](https://nf-co.re/differentialabundance) workflow.
+
+We recommend that you provide reference files directly, via `--gtf` and `--fasta`, and that supplied GTF files do not use gene names as `gene_id`.
```

Comment:
Probably also worth extending these to provide genome indices too. And maybe even docs and commands to download via Ensembl. I am manually sharing them right now on Slack which isn't ideal.

Reply:
@drpatelh I've added the example download commands, and reworked a bit to make the existing discussion of indices more prominent.